docs/source/markdown/podman-kube-generate.1.md

```diff
@@ -6,33 +6,33 @@ podman-kube-generate - Generate Kubernetes YAML based on containers, pods or vol
 
 Potential name conflicts between volumes are avoided by using a standard naming scheme for each volume type. The *hostPath* volume types are named according to the path on the host machine, replacing forward slashes with hyphens less any leading and trailing forward slashes. The special case of the filesystem root, `/`, translates to the name `root`. Additionally, the name is suffixed with `-host` to avoid naming conflicts with *persistentVolumeClaim* volumes. Each *persistentVolumeClaim* volume type uses the name of its associated named volume suffixed with `-pvc`.
 
-Note that if an init container is created with type `once` and the pod has been started, the init container will not show up in the generated kube YAML as `once` type init containers are deleted after they are run. If the pod has only been created and not started, it will be in the generated kube YAML.
-Init containers created with type `always` will always be generated in the kube YAML as they are never deleted, even after running to completion.
+Note that if an init container is created with type `once` and the pod has been started, the init container does not show up in the generated kube YAML as `once` type init containers are deleted after they are run. If the pod has only been created and not started, it is in the generated kube YAML.
+Init containers created with type `always` are always generated in the kube YAML as they are never deleted, even after running to completion.
 
 *Note*: When using volumes and generating a Kubernetes YAML for an unprivileged and rootless podman container on an **SELinux enabled system**,  one of the following options must be completed:
   * Add the "privileged: true" option to the pod spec
   * Add `type: spc_t` under the `securityContext` `seLinuxOptions` in the pod spec
   * Relabel the volume via the CLI command `chcon -t container_file_t -R <directory>`
 
-Once completed, the correct permissions will be in place to access the volume when the pod/container is created in a Kubernetes cluster.
+Once completed, the correct permissions are in place to access the volume when the pod/container is created in a Kubernetes cluster.
 
 Note that the generated Kubernetes YAML file can be used to re-run the deployment via podman-play-kube(1).
 
 ## OPTIONS
 
 #### **--filename**, **-f**=*filename*
 
-Output to the given file, instead of STDOUT. If the file already exists, `kube generate` will refuse to replace it and return an error.
+Output to the given file, instead of STDOUT. If the file already exists, `kube generate` refuses to replace it and return an error.
```

Comment:
```suggestion
Output to the given file instead of STDOUT. If the file already exists, `kube generate` refuses to replace it and returns an error.
```